cli/src/formatter.rs

```diff
@@ -272,6 +272,8 @@ pub struct ColorFormatter<W: Write> {
     cached_styles: HashMap<Vec<String>, Style>,
     /// The style we last wrote to the output.
     current_style: Style,
+    /// The debug string (space-separated labels) we last wrote to the output.
+    current_debug: String,
```

Comment:
nit: for extra compile-time guarantee, maybe we can combine `debug: bool` and `current_debug: String` as `Option<String>` or `current_debug_labels: Option<Vec<String>>`?